app/views/shared/_field_with_error.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  label ||= ''
+  type ||= :text_field
+  qa ||= ''
+  select_options ||= []
+  input_options = {
```

Comment:
I'd prefer if we were passing in an input to this partial. Probably need some sort of helper for the aria-describedby attributes.